src/cli/init_command.zig

```diff
@@ -384,9 +387,20 @@ pub const InitCommand = struct {
                     auto_yes = true;
                 } else if (strings.eqlComptime(arg, "--")) {
                     parse_flags = false;
+                } else if (strings.eqlComptime(arg, "--react") or strings.eqlComptime(arg, "-r")) {
+                    template = .react_blank;
+                    prev_flag_was_react = true;
+                    auto_yes = true;
+                } else if ((template == .react_blank and prev_flag_was_react and strings.eqlComptime(arg, "tailwind") or strings.eqlComptime(arg, "--react=tailwind")) or strings.eqlComptime(arg, "r=tailwind")) {
+                    template = .react_tailwind;
+                    auto_yes = true;
+                } else if ((template == .react_blank and prev_flag_was_react and strings.eqlComptime(arg, "shadcn") or strings.eqlComptime(arg, "--react=shadcn")) or strings.eqlComptime(arg, "r=shadcn")) {
+                    template = .react_tailwind_shadcn;
+                    auto_yes = true;
                 } else {
                     // invalid flag; ignore
                 }
+                prev_flag_was_react = false;
```

Comment:
this will always set prev_flag_was_react to false. should it be in the else block?